### PR TITLE
Setup prettier to do checking and formatting

### PR DESCRIPTION
**Pull request by MatrixFrog**
Fixes https://github.com/distributeaid/next-website-v2/issues/93

### CODE REVIEW

**Comment by MatrixFrog** [comment]
@jtfairbank this creates a lot of little changes (looks like mostly just whitespace/semicolons/etc) but I guess it's good to have the formatting be consistent

File: src/app/resources/assort/assort.tsx [hunk 1]
```diff
@@ -1,64 +1,156 @@
...
+                How to box and label the sorted donations and ready them for
+                shipment.
+              </li>
+            </ul>
+          </article>
+          <article className="border-l-2 border-navy-400 pl-4 mt-8">
+            <p>
+              ASSORT was established as part of a joint partnership between
+              <span>
+                {" "}
```
Comment (MatrixFrog) [inline src/app/resources/assort/assort.tsx:71]: i think we maybe want to clean up some of these spaces

**Comment by jtfairbank** [comment]
@MatrixFrog yeah formatting a repo for the first time always does :sweat_smile:

File: .prettierignore [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.github/
+.yarn/
+.yarnrc.yml
+.gitpod.yml
+.prettierrc
+next-env.d.ts
+README.md
+renovate.json
```
Comment (jtfairbank) [inline .prettierignore:8]: @MatrixFrog **question:** Is there a reason to exclude the dot files and readme and such?  On the API project it looks like it's mostly just the auto-generated files.

https://github.com/distributeaid/aggregated-public-information/blob/saga/.prettierignore

File: src/app/resources/assort/assort.tsx (hunk 1 above)
Comment (jtfairbank) [reply src/app/resources/assort/assort.tsx:71]: @MatrixFrog sure thing!  What did you have in mind instead?  want to include that in this PR?

Comment (MatrixFrog) [reply src/app/resources/assort/assort.tsx:71]: I think let's do a separate one just to get rid of any unnecessary spaces, and then do this one after.

File: .prettierignore (hunk 2 above)
Comment (MatrixFrog) [reply .prettierignore:8]: I guess that's true. the next-env file says it shouldn't be edited but anything that's JSON is probably fine to format.

File: src/app/resources/assort/assort.tsx (hunk 1 above)
Comment (MatrixFrog) [reply src/app/resources/assort/assort.tsx:71]: actually after looking closer, most of these spaces are necessary, let's just go forward with this

**Pull request merged**